webapp/components/Registration/CreateUserAccount.spec.js

```diff
@@ -81,7 +82,7 @@ describe('CreateUserAccount', () => {
               email: 'sixseven@example.org',
               nonce: '666777',
               password: 'hellopassword',
-              termsAndConditionsAgreedVersion: '0.0.2',
+              termsAndConditionsAgreedVersion: VERSION,
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-24T11:46:51Z" title="Thursday, October 24th 2019, 1:46:51 pm +02:00">Oct 24, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -81,7 +82,7 @@ describe('CreateUserAccount', () => {
               email: 'sixseven@example.org',
               nonce: '666777',
               password: 'hellopassword',
-              termsAndConditionsAgreedVersion: '0.0.2',
+              termsAndConditionsAgreedVersion: VERSION,
```


Yesterday in the lecture I told the students, that hard-coded values in the tests are OK :laughing: I think in this very situation it's fine to import the `VERSION` constant